src/Servers/IIS/IIS/test/IIS.LongTests/IIS.LongTests.csproj

```diff
@@ -0,0 +1,53 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <TestGroupName>IIS.LongTests</TestGroupName>
+    <DefineConstants>IIS_FUNCTIONALS</DefineConstants>
+    <DisableFastUpToDateCheck>True</DisableFastUpToDateCheck>
+    <SkipTests Condition=" '$(SkipIISTests)' == 'true' ">true</SkipTests>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <Import Project="../FunctionalTest.props" />
+
+  <ItemGroup>
+    <ProjectReference Include="..\testassets\IIS.Common.TestLib\IIS.Common.TestLib.csproj" />
+    <ProjectReference Include="..\testassets\InProcessWebSite\InProcessWebSite.csproj"
+      Private="false"
+      ReferenceOutputAssembly="false"
+      SkipGetTargetFrameworkProperties="true" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Common.FunctionalTests\Infrastructure\**\*.cs" />
+    <Compile Include="..\Common.LongTests\**\*.cs" />
+    <Compile Include="$(SharedSourceRoot)Http2cat\**\*.cs" LinkBase="Shared\Http2cat" />
+    <Compile Include="$(SharedSourceRoot)runtime\Http2\**\*.cs" LinkBase="Shared\Http2\" />
+    <Compile Include="$(SharedSourceRoot)runtime\IHttpHeadersHandler.cs" LinkBase="Shared\IHttpHeadersHandler.cs" />
+    <Compile Include="$(SharedSourceRoot)runtime\SR.cs" LinkBase="Shared\SR.cs" />
+    <Compile Include="$(SharedSourceRoot)ServerInfrastructure\**\*.cs" LinkBase="Shared\" />
+    <Compile Include="$(SharedSourceRoot)ValueTaskExtensions\**\*.cs" LinkBase="Shared\" />
+    <Compile Remove="$(SharedSourceRoot)ServerInfrastructure\DuplexPipe.cs" />
```

Comment:
Suggest using `Exclude` in the `$(SharedSourceRoot)ServerInfrastructure\**\*.cs` items. At least, please move this next to that line.

Reply:
What's the benefit of exclude vs remove, if its functionally equivalent I'd rather not make changes for no benefit, either way I'll tweak the ordering so they are together though

Reply:
`Exclude` is very helpful both in terms of readability and avoiding excess logging. `Remove` for cases like this doesn't have any advantages and bloats the binary log files we create.

Reply:
Seems subjective, but I'll switch these to Exclude to make you happy @dougbu

Reply:
Much appreciated❕ I live to make me happy too 😄